scripts/release/monitor/monitor_release/portainer_utils.py

```diff
@@ -0,0 +1,110 @@
+import json
+
+import arrow
+import requests
+from settings import Settings
+
+
+def get_bearer_token(settings: Settings):
+    headers = {"accept": "application/json", "Content-Type": "application/json"}
+    payload = json.dumps(
+        {
+            "Username": settings.portainer_username,
+            "Password": settings.portainer_password,
+        }
+    )
+    response = requests.post(
```

Comment:
I would use the libraries we normally use instead of adding new ones e.g.  prefer `httpx` to `request`